cl/people_db/models.py

```diff
@@ -1432,6 +1592,340 @@ def save(self, *args, **kwargs):
+
+    financial_disclosure = models.ForeignKey(
+        FinancialDisclosure,
+        help_text="The financial disclosure associated with this investment.",
+        related_name="non_investment_income",
+        on_delete=models.CASCADE,
+    )
+    date_created = models.DateTimeField(
+        help_text="The original creation date for the item",
+        auto_now_add=True,
+        db_index=True,
+    )
+    date_modified = models.DateTimeField(
+        help_text="The last moment when the item was modified.",
+        auto_now=True,
+        db_index=True,
+    )
+    date = models.TextField(
+        help_text="Date of non-investment income (ex. 2011).",
+        blank=True,
+    )
+    source_type = models.TextField(
+        help_text="Source and type of non-investment income for the judge "
+        "(ex. Teaching a class at U. Miami).",
+        blank=True,
+    )
+    income_amount = models.TextField(
+        help_text="Amount earned by judge.",
+        blank=True,
+    )
+    redacted = models.BooleanField(
+        help_text="Is non-investment income redacted?",
+        default=False,
+    )
+
+
+class SpouseIncome(models.Model):
+    """ Financial Disclosure Judge Spouse Income Table"""
+
+    financial_disclosure = models.ForeignKey(
+        FinancialDisclosure,
+        help_text="The financial disclosure associated with this investment.",
+        related_name="spouse_income",
+        on_delete=models.CASCADE,
+    )
+    date_created = models.DateTimeField(
+        help_text="The original creation date for the item",
+        auto_now_add=True,
+        db_index=True,
+    )
+    date_modified = models.DateTimeField(
+        help_text="The last moment when the item was modified.",
+        auto_now=True,
+        db_index=True,
+    )
+    source_type = models.TextField(
+        help_text="Source and Type of income of judicial spouse",
+        blank=True,
+    )
+    date = models.TextField(
+        help_text="Date of spousal income (ex. 2011).",
+        blank=True,
+    )
+    redacted = models.TextField(
+        help_text="Is judicial spousal income redacted?",
+        default=False,
+    )
+
+
+class Reimbursement(models.Model):
+    """Reimbursements listed in judicial disclosure"""
+
+    financial_disclosure = models.ForeignKey(
+        FinancialDisclosure,
+        help_text="The financial disclosure associated with this investment.",
+        related_name="reimbursement",
+        on_delete=models.CASCADE,
+    )
+    date_created = models.DateTimeField(
+        help_text="The original creation date for the item",
+        auto_now_add=True,
+        db_index=True,
+    )
+    date_modified = models.DateTimeField(
+        help_text="The last moment when the item was modified.",
+        auto_now=True,
+        db_index=True,
+    )
+    source = models.TextField(
+        help_text="Source of the reimbursement (ex. FSU Law School).",
+        blank=True,
+    )
+    dates = models.TextField(
```

Comment:
dates is plural here - because in this instance they almost always provide a date range for the conferences or whatever.  for example (January 16-17, 2014).